src/store/cache-store.ts

```
@@ -0,0 +1,114 @@
+        const { inited, ...rest } = state;
+        return rest;
+      },
+      name: LOCAL_STORAGE_KEY,
+      version: STORAGE_VERSION,
+      migrate: (persistedState, version) => {
+        if (version < STORAGE_VERSION) {
+          // Handle migration or reset state
+          return {
+            inited: false,
+            attestations: {}
+          } as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)